### PR TITLE
explicitly use ast namespace when including files

### PR DESCRIPTION
**Pull request by chantra**
##### Checklist

- [x] The new behaviour is covered by tests

##### Why

a couple of files from ast directory were included using the root namespace. To make this work, CMakeLists.txt needed to explicitely:
```
target_include_directories(runtime PRIVATE ${CMAKE_SOURCE_DIR}/src/ast)
```

This change update the couple of .cpp files that included ast header files from the root namespace to use the ast prefix, and remove the `target_include_directories` entries.

While it does not change anything fundamentally to how bpftrace is build using cmake, it does make building with other build systems such as buck, bazel... easier.

### CODE REVIEW

**Review by jordalgo — APPROVED** [review]
Thanks for the fix. LGTM!

**Comment by chantra** [comment]
got to fix a few lint errors...

File: src/output.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #include "bpftrace.h"
 #include "log.h"
 #include "utils.h"
-#include <async_event_types.h>
+#include <ast/async_event_types.h>
```
Comment (viktormalik) [inline src/output.cpp:5]:
```suggestion
#include "ast/async_event_types.h"
```
and order alphabeticlaly

**Review by viktormalik — COMMENTED** [review]
Nice change, thanks!

**Review by danobi — APPROVED** [review]
clang-format doesn't look quite right... maybe ignore it in this case

**Comment by chantra** [comment]
> clang-format doesn't look quite right... maybe ignore it in this case

@danobi are you referring to the fact that "output.h" was put at the top? https://github.com/iovisor/bpftrace/actions/runs/7629941448/job/20784517389#step:8:10

I think it does make sense. It signals that the files' `.h` should be first. I the updated diff, I have added an empty line between "output.h" and the block of includes so sorting is done within the block.

**Comment by danobi** [comment]
D'oh, didn't realize that was in output.cpp. Tool is pretty clever :)

**Pull request merged**